server/appbackend/urls.py

```diff
@@ -1,14 +1,15 @@
 from django.conf.urls import *
 from django.urls import path
 from rest_framework import routers
-from .api import TurmaApi, TurmasDoAlunoApi, CourseApi, TurmaDetailApi, StudentApi, StudentDetailApi
-
+from .api import *
 urlpatterns = [
-    path('turmas/', TurmaApi.as_view(), name='turma-list'),
-    path('turmas/<int:pk>', TurmaDetailApi.as_view(), name='turma-detail'),
+    path('turmas/', ClassGroupApi.as_view(), name='turma-list'),
+    path('turmas/<int:pk>', ClassGroupDetailApi.as_view(), name='turma-detail'),
     path('students/', StudentApi.as_view(), name='student-list'),
     path('students/<int:pk>/', StudentDetailApi.as_view(), name='student-detail'),
-    path('students/<int:pk>/turmas', TurmasDoAlunoApi.as_view(), name='student-turmas-list')
+    path('students/<int:pk>/class-groups', StudentClassGroupsApi.as_view(), name='student-turmas-list'),
```

Comment:
<img width="470" alt="Screen Shot 2019-05-14 at 21 25 37" src="https://user-images.githubusercontent.com/13035425/57740582-f0dd9d80-768e-11e9-95e2-80b69814d1c8.png">

Reply:
Endpoint not working till 'enrollments' is resolved